views/attendance/attendance_edit_class_info.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+/**
+ * PEP Capping 2017 Algozzine's Class
+ *
+ * enter information to take attendance for new class
+ *
+ * provides a list of fields to specify what class attendance will be recorded for
+ *
+ * @author Scott Hansen
+ * @copyright 2017 Marist College
+ * @version [version number]
+ * @since [initial version number]
+ */
+
+authorizedPage();
+
+global $db;
+
+include ('attendance_utilities.php');
+
+//form options
+$result_curriculum = $db->no_param_query("SELECT c.curriculumid, c.curriculumname FROM curricula c ORDER BY c.curriculumname ASC;");
```

Comment:
Make sure to only get current curricula:

```sql
SELECT c.curriculumid, c.curriculumname
FROM curricula c
WHERE c.df IS FALSE
ORDER BY c.curriculumname ASC
```

Reply:
need all of them for creating my `classesMatrix` which contains all of the topic names, the id's for those topics, and the curriculum they belong to.

Reply:
`IS FALSE` results in a syntax error, but I can do `c.df = 0`